common/djangoapps/util/html.py

```diff
@@ -14,6 +14,7 @@ def __init__(self):
         """
         Initializer
         """
+        HTMLParser.__init__(self)
```

Comment:
can we use super(MLStripper, self).__init__() here? considering that is the better approach. 
just to know about your thoughts about that.